lang/en.lyaml

```diff
@@ -1249,6 +1259,13 @@ en:
           describe: "Use environment variable config"
         debug:
           describe: "Set log level to debug"
+    configMigrate:
+      migrateConfigPrompt: "Migrate config?\n From deprecated config file: {{ deprecatedConfigPath }}\n To new global config file: {{ globalConfigPath }}\n (This action will delete the deprecated config file).\n"
+      mergeConfigConflictPrompt: "Overwrite the conflicting property in the global config?\n {{ property }}: {{ values }} "
```

Comment:
You could shorten this by doing:
`Change the {{ property }} property in the global config from {{old}} to {{new}}?`

Which would show as:
`Change the httpTimeout property in the global config from 2000 to 3000? Y/n`

Reply:
I'll ask Jono--this was placeholder text, since he was OOO on Thursday/Friday.